docs/glossary.rst

```diff
@@ -149,21 +149,20 @@ authentication for devices. We recommend using one of:
 
 Transfer Device
 ---------------
-
-The *Transfer Device* is the physical media used to transfer encrypted
-documents from the *Journalist Workstation* to the *Secure Viewing
-Station*. Examples: a dedicated USB stick, CD-R, DVD-R, or SD card.
-
-If you use a USB stick for the *Transfer Device*, we recommend using a
-small one (4GB or less). It will be necessary to securely wipe the entire
-device at times, and this process takes longer for larger devices.
-
-Depending on your threat model, you may wish to only use one-time-use
-media (such as CD-R or DVD-R) for transferring files to and from the
-*SVS*. While doing so is cumbersome, it reduces the risk of malware (that
-could be run simply by opening a malicious submission) exfiltrating
-sensitive data, such as the private key used to decrypt submissions or
-the content of decrypted submissions.
-
-When we use the phrase "sneakernet" we mean physically moving documents
-with the *Transfer Device* from one computer to another.
+The *Transfer Device* is the physical media (e.g., designated USB drive) used
+to  transfer encrypted documents from the *Journalist Workstation* to the
+*Secure Viewing Station*. Both computers run the Tails operating system
```

Comment:
[Nit/typo: there are 2 spaces in "to transfer"]

I would take out the sentence about "both computers run.." because this is just the definition of the transfer device.

"The *Transfer Device* is the physical media (e.g., designated USB drive) used to transfer encrypted documents from the *Journalist Workstation* to the *Secure Viewing Station*, where they can be decrypted."